salsa_dancing_molecules/scatter_plot.py

```diff
@@ -12,8 +12,48 @@ def draw_scatter_plot(x_axis, y_axis, xlabel, ylabel, filename):
     File is saved as filename (which should include .png).
     """
     # Agg -> use non-interactive backend to save plot to file
-    matplotlib.use('Agg')
-    plt.scatter(x_axis, y_axis)
+    # matplotlib.use('Agg')
```

Comment:
This was added to allow this to be run on a system without interactivity, such as the super computer. I don't think you should comment this line out.